### PR TITLE
chore: use different metric default name

### PR DESCRIPTION
**Pull request by yquansah**
Signed-off-by: Yoofi Quansah <yqsummon@gmail.com>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] Tests have been added
- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [ ] Changelog has been updated

Fixes #

### CODE REVIEW

File: pkg/scalers/influxdb_scaler.go [hunk 1]
```diff
@@ -103,6 +97,16 @@ func parseInfluxDBMetadata(config *ScalerConfig) (*influxDBMetadata, error) {
 		return nil, fmt.Errorf("no server url given")
 	}
 
+	if val, ok := config.TriggerMetadata["metricName"]; ok {
```
Comment (zroubalik) [inline pkg/scalers/influxdb_scaler.go:100]: Could you please take a look on how Postrgresql scaler is done? We need to mask certain characters in the metricName:
 https://github.com/kedacore/keda/blob/cfebd9e0384c54f23caaeb0074866d5685f0c77a/pkg/scalers/postgresql_scaler.go#L118-L131

For influxdb I think you can use the combination of `prefix (infludb-) + url + organisationName`

**Review by zroubalik — CHANGES_REQUESTED** [review]

**Review by zroubalik — APPROVED** [review]
LGTM, thanks @yquansah !

**Comment by zroubalik** [comment]
@yquansah there's a problem in the linter :(

**Comment by yquansah** [comment]
@zroubalik Addressed

**Review by zroubalik — APPROVED** [review]
LGTM, thanks @yquansah !

**Pull request merged**